plugins/repository-s3/qa/amazon-s3/src/test/java/org/elasticsearch/repositories/s3/AmazonS3Fixture.java

```diff
@@ -53,8 +53,8 @@
 public class AmazonS3Fixture {
 
     public static void main(String[] args) throws Exception {
-        if (args == null || args.length != 2) {
-            throw new IllegalArgumentException("AmazonS3Fixture <working directory> <bucket>");
+        if (args == null || args.length != 1) {
```

Comment:
Can we revert this please?

Reply:
We now pass the bucket names in on the command line again.